src/xunit.performance.execution/BenchmarkEventSource.cs

```diff
@@ -191,14 +201,24 @@ public unsafe void BenchmarkIterationStop(string RunId, string BenchmarkName, in
                 }
             }
 
-            if (_csvWriter != null)
+            if (s_csvWriter != null)
                 WriteCSV(BenchmarkName, iteration: Iteration);
         }
 
-        public static readonly BenchmarkEventSource Log = new BenchmarkEventSource();
+        private BenchmarkEventSource() : base(true)
```

Comment:
In the case of coreclr perf tests, the API did not throw an exception, but `EventSource.IsEnable` always returned false.